WooCommerce/Classes/Authentication/Jetpack Setup/Native Jetpack Setup/JetpackSetupViewModel.swift

```diff
@@ -223,7 +232,11 @@ private extension JetpackSetupViewModel {
             switch result {
             case .success(let url):
                 self.analytics.track(.loginJetpackSetupFetchJetpackConnectionURLSuccessful)
-                self.jetpackConnectionURL = url
+                if url.absoluteString.hasPrefix(Constants.accountConnectionURL) {
+                    self.jetpackConnectionURL = url
+                } else {
+                    self.jetpackConnectionURL = self.siteConnectionURL
+                }
```

Comment:
Super nit:

This is probably because I lack knowledge about the project. But I don't understand what is `siteConnectionURL` and why we prefer that over the `url` in this scenario.

Please consider adding comments. Or, if it makes sense, we could move this logic to a private helper to explain better (by taking advantage of the method name)

Reply:
Added comment here e86ceb95948d17f1970e5d166d9dcdc286a79e6a.